src/components/form/form.tsx

```diff
@@ -118,13 +119,34 @@ const fields: Array<FieldInterface> = [
 
 export default function Form({
   isOpen,
-  onClose,
+  closeForm,
 }: FormProps): JSX.Element | null {
+  const [formError, setFormError] = useState<boolean>(false);
+  const [formSuccess, setFormSuccess] = useState<boolean>(false);
+
   async function onSubmit(form: FormValues): Promise<void> {
+    // clear out existing error state if present
+    if (formError) {
+      setFormError(false);
+    }
+
     const body = form as unknown as Registration; // converts required formValues arg type to required Registration type for POST
-    // Temporary Implementation
-    await uploadToAppServices(body);
-    return;
+
+    try {
+      const req = await axios.post('/academia/api/registration', body);
+
+      if (req.status === 200) {
+        setFormSuccess(true);
```

Comment:
This seems like it might be redundant since `axios.post` will resolve if it's a 2xx status.